.env

```diff
@@ -26,3 +26,4 @@ COLLECT_YEAR_OF_BIRTH=true
 APP_ID=''
 MFE_CONFIG_API_URL=''
 ENABLE_SKILLS_BUILDER=''
+ENABLE_LEARNING_GOAL=''
```

Comment:
I have one additional tiny nit to suggest -- what do you think of updating the name of this feature flag to something like 'ENABLE_SKILLS_BUILDER_PROFILE' to control displaying skills-builder specific information on the profile? This could eventually include more data beyond the learning goal... so maybe we update it to be more general? 

Also, there is a "goal" field in the user profile model that isn't displayed here... so it could be potentially confusing?

Reply:
Totally agree on this, I'll make the change!